Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM usgswma/python:3.8
 
-RUN apt-get install --no-install-recommends -y curl git
+RUN apt-get install --no-install-recommends -y curl git dnsutils
 RUN curl -sL https://deb.nodesource.com/setup_13.x | bash - && apt-get install -y nodejs
```

Comment:
We should probably switch this over to version 12.

Reply:
oh thanks for catching!